src/redux/reducers/currentUserReducer.js

```diff
@@ -0,0 +1,19 @@
+import { SET_CURRENT_USER } from '../actions-types';
+
+const initialState = {
+  isLoggedIn: false,
+  profile: {},
+  articles: [],
+  favorites: [],
+};
+
+const currentUserReducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+    case SET_CURRENT_USER:
+      return { ...state, profile: payload, isLoggedIn: true };
+    default:
+      return { ...state };
```

Comment:
there's no need for braces, you can just write `return ...state` since its a single return